torchao/testing/utils.py

```diff
@@ -98,6 +98,47 @@ def wrapper(*args, **kwargs):
     return decorator
 
 
+def skip_if_no_xpu(message=None):
```

Comment:
currently these things are a bit confusing I think, since I remember some of these work with unittests and others work with pytest. can you confirm what these are working with? I think ideally we should create two versions, one for unittests one for pytest